package.json

```diff
@@ -52,11 +52,11 @@
 		"nanoid": "^3.1.23",
 		"react-dropzone": "10.2.0",
 		"react-inlinesvg": "^2.3.0",
-		"react-pdf": "^4.1.0",
+		"react-pdf": "^5.3.2",
 		"react-select": "^4.3.1"
 	},
 	"peerDependencies": {
-		"react": ">=16.2",
+		"react": "^17.0.2",
```

Comment:
We need Phoenix Components to be compatible also with React 16 because that's what is currently used in MyAxiory. Please try something like `>=16.2 <= 17` or `^16.2.0 || ^17.0.0`